generated/nidaqmx/errors.py

```diff
@@ -45,18 +45,24 @@ def __init__(self, message, error_code, task_name=''):
             message (string): Specifies the error message.
             error_code (int): Specifies the NI-DAQmx error code.
         """
-        if task_name:
-            message = f'{message}\n\nTask Name: {task_name}'
-
-        super().__init__(message)
-
         self._error_code = int(error_code)
 
         try:
             self._error_type = DAQmxErrors(self._error_code)
         except ValueError:
             self._error_type = DAQmxErrors.UNKNOWN
 
+        # If message is empty, we try to put at least some information in it
+        if not message and self._error_type != DAQmxErrors.UNKNOWN:
+            message = f'Explanation could not be found for the requested status code.\n\nVerify that the requested status code is correct.\n\nError {self._error_type.name}'
+
+        if task_name:
+            message = f'{message}\n\nTask Name: {task_name}'
+            
+        message = f'{message}\n\nStatus Code: {self._error_code}'
```

Comment:
The C API already adds "Status Code:". This code should only add it when the message didn't come from the C API.